### PR TITLE
handle case where mms could not be retrieved

### PR DESCRIPTION
**Pull request by lduboeuf**
Pre-requisite: https://github.com/ubports/nuntium/pull/3

### CODE REVIEW

File: indicator/textchannelobserver.cpp [hunk 1]
```diff
@@ -499,6 +499,9 @@ void TextChannelObserver::showNotificationForMessage(const Tp::TextChannelPtr ch
             messageText = QString::fromUtf8(C::ngettext("Attachment: %1 audio clip", "Attachments: %1 audio clips", audioCount)).arg(audioCount);
         } else if (attachmentCount > 0) {
             messageText = QString::fromUtf8(C::ngettext("Attachment: %1 file", "Attachments: %1 files", attachmentCount)).arg(attachmentCount);
+        } else if (attachmentCount == 0) {
```
Comment (Flohack74) [inline indicator/textchannelobserver.cpp]: You have before attahcmentCount > 0, so now the elseif can be just else or? There will be no sane way that attachmentcount is smaller than 0 :) -so please make it unconditional else only.

File: indicator/textchannelobserver.cpp [hunk 2]
```diff
@@ -499,6 +499,9 @@ void TextChannelObserver::showNotificationForMessage(const Tp::TextChannelPtr ch
...
+            // TRANSLATORS : message displayed when any error occured while receiving a mms ( case when cellular-data is off, or any downloading issue). Note that mms is lost in that case
+            messageText = QString::fromUtf8(C::gettext("Oops, there has been an error with the MMS system and this message could not be retrieve. Please ensure Cellular Data is ON and MMS settings are correct, then ask the sender to try again."));
```
Comment (Flohack74) [inline indicator/textchannelobserver.cpp]: typo: retrieveD

**Review by Flohack74 — CHANGES_REQUESTED** [review]

**Comment by lduboeuf** [comment]
@Flohack74 note that there is no translation up in weblate for telephony-service afaik

**Review by Flohack74 — APPROVED** [review]

**Comment by Flohack74** [comment]
Ok we will put smth on Weblate thx!

**Review by Flohack74 — APPROVED** [review]

**Comment by jezek** [comment]
Device: FP2
OS: 2020-W49

On failed MMS deliver*, in the notification bubble there is a text "Oops, ...". Clicking on the bubble opens ```messaging-app``` focused on the sender conversation's latest message, which (in this failed case) is the "Oops, ..." text with red exclamation mark (:exclamation: ) in circle + vertical stripe on the left of the text.

Edit: Works as intended.

**Pull request merged**